app/commands.py

```diff
@@ -713,12 +717,23 @@ def boolean_or_none(field):
         for line in itertools.islice(f, 1, None):
             columns = line.split('|')
             print(columns)
+            email_branding = None
+            email_branding_column = columns[5]
```

Comment:
Could do `columns[5].strip()` like you've done with letter branding below just in case the column contains whitespace